sydent/http/servlets/store_invite_servlet.py

```diff
@@ -79,19 +79,22 @@ def render_POST(self, request):
         tokenStore.storeEphemeralPublicKey(ephemeralPublicKeyBase64)
         tokenStore.storeToken(medium, address, roomId, sender, token)
 
+        # Variables to substitute in the template.
         substitutions = {}
+        # Include all arguments sent via the request.
         for k, v in args.items():
             if isinstance(v, string_types):
                 substitutions[k] = v
         substitutions["token"] = token
 
+        # Additional arguments that are optional, but the template might expect.
         required = [
```

Comment:
> Additional arguments that are optional

> variable is named `required`

This is slightly confusing. Perhaps we could rename `required` to `extra_substitutions` or something.

Reply:
Good call. 👍